docs/manual/linux-udev.md

```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Configuring udev on Linux
 parent: User manual
-nav_order: 5
+nav_order: 7
```

Comment:
shouldn't touch user manual if you're only changing porting docs

Reply:
Is because this:

1 first-use
2 layers
3 macros
4 tap-dance
5 combos
5 udev
6 QMK settings